tests/_support/Page/Acceptance/CreateBookChapter.php

```diff
@@ -0,0 +1,35 @@
+<?php
+namespace Page\Acceptance;
+
+use AcceptanceTester;
+
+class CreateBookChapter
+{
+	public static $URL = '/wp-admin/post-new.php?post_type=chapter';
+
+	protected $titleField = '#title';
+	protected $contentField = '#content';
+	protected $saveButton = '#publishing-action input[type=submit]';
+
+	/**
+	 * @var AcceptanceTester;
+	 */
+	protected $acceptanceTester;
+
+	public function __construct(AcceptanceTester $I)
+	{
+		$this->acceptanceTester = $I;
+	}
+
+	public function createChapter(string $bookURL, string $title, string $content = null): void
+	{
+		$I = $this->acceptanceTester;
+
+		$I->amOnPage( $bookURL . self::$URL);
+		$I->fillField($this->titleField, $title);
+		$I->fillField( $this->contentField, $content );
+		// Figure it out how to enable tinymce when running tests
+		// $I->fillTinyMceEditorByName('content', $content);
```

Comment:
I'm filling the default content field because TinyMce is not enabled at the moment. Need to research a bit more on how to enable it.